Test/lambdas/LitInt.dfy

```diff
@@ -0,0 +1,22 @@
+// RUN: %dafny /dprint:"%t.dprint" "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+const N: nat
+
+function Fibonacci(n: int): int
+{
+	if n <= 1 then 1
```

Comment:
This is not the Fibonacci I know, because it skips 0. I suggest:

    if n < 2 then n